packages/main/src/StepInput.js

```diff
@@ -0,0 +1,540 @@
+
+		/**
+		 * Determines the name with which the <code>ui5-step-input</code> will be submitted in an HTML form.
+		 *
+		 * <br><br>
+		 * <b>Important:</b> For the <code>name</code> property to have effect, you must add the following import to your project:
+		 * <code>import "@ui5/webcomponents/dist/features/InputElementsFormSupport.js";</code>
+		 *
+		 * <br><br>
+		 * <b>Note:</b> When set, a native <code>input</code> HTML element
+		 * will be created inside the <code>ui5-step-input</code> so that it can be submitted as
+		 * part of an HTML form. Do not use this property unless you need to submit a form.
+		 *
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 */
+		name: {
+			type: String,
+		},
+
+		/**
+		 * Determines the text alignment of the <code>ui5-step-input</code>.
+		 *
+		 * @type {string}
+		 * @defaultvalue "left"
+		 * @public
+		 */
+		textAlign: {
+			type: String,
+			defaultValue: "left",
+		},
+
+		/**
+		 * Determines the number of digits after the decimal point of the <code>ui5-step-input</code>.
+		 *
+		 * @type {Integer}
+		 * @defaultvalue 0
+		 * @public
+		 */
+		valuePrecision: {
+			type: Integer,
+			defaultValue: 0,
+		},
+
+		/**
+		 * Defines the aria-label attribute for the <code>ui5-step-input</code>.
+		 *
+		 * @type {String}
+		 * @since 1.0.0-rc.9
+		 * @private
+		 * @defaultvalue ""
+		 */
+		ariaLabel: {
+			type: String,
+		},
+
+		/**
+		 * Receives id(or many ids) of the elements that label the <code>ui5-step-input</code>.
+		 *
+		 * @type {String}
+		 * @defaultvalue ""
+		 * @private
+		 * @since 1.0.0-rc.9
+		 */
+		ariaLabelledby: {
+			type: String,
+			defaultValue: "",
+		},
+
+		_decIconDisabled: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		_incIconDisabled: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		_focused: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		_inputFocused: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		_previousValue: {
+			type: Float,
+			noAttribute: true,
+		},
+
+	},
+	slots: /** @lends sap.ui.webcomponents.main.StepInput.prototype */ {
+		/**
+		 * Defines the value state message that will be displayed as pop up under the <code>ui5-step-input</code>.
+		 * <br><br>
+		 *
+		 * <b>Note:</b> If not specified, a default text (in the respective language) will be displayed.
+		 * <br>
+		 * <b>Note:</b> The <code>valueStateMessage</code> would be displayed,
+		 * when the <code>ui5-step-input</code> is in <code>Information</code>, <code>Warning</code> or <code>Error</code> value state.
+		 * @type {HTMLElement}
+		 * @since 1.0.0-rc.7
+		 * @slot
+		 * @public
+		 */
+		valueStateMessage: {
+			type: HTMLElement,
+		},
+	},
+	events: /** @lends sap.ui.webcomponents.main.StepInput.prototype */ {
+		/**
+		 * Fired when the input operation has finished by pressing Enter or on focusout.
+		 *
+		 * @event
+		 * @public
+		*/
+		change: {},
+	},
+};
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ *
```

Comment:
Here you should add an overview what is this component about + since tag (use RC.12)

Reply:
Added